src/pages/reset-password/index.tsx

```diff
@@ -133,7 +140,7 @@ const ResetPassword = () => {
                                 />
                             </InputGroup>
                             <Text align="center" color="red">
-                                {status.error}
+                                {apiError ? apiError : status.error}
```

Comment:
```suggestion
                                {apiError || status.error}
```

Reply:
Hi @mayuran-deriv thanks for pointing this. 🙏 
Is there any performance difference between both techniques ?

Reply:
hi, @fasihali-deriv 
there shouldn't be a significant difference between the two expressions. but apiError || status.error expression is shorter and more idiomatic, which can enhance code readability.